src/stores/DeviceStore.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const alt = require('../alt');
+
+const DeviceActions = require('../actions/DeviceActions.js');
+
+class DeviceStore {
+  constructor() {
+
+    this.bindActions(DeviceActions);
```

Comment:
also, if you ever want to remap actions to functions, explicit binding wins by a long shot